src/css/dark.css

```diff
@@ -110,6 +109,7 @@ input.number:focus {
 		}
 
 		#menubar .menu .options {
+      cursor: pointer;
```

Comment:
Hehe, fixed the whitespace in https://github.com/aframevr/aframe-editor/pull/88 :P